### PR TITLE
Improved PCL support

### PR DESCRIPTION
**Pull request by superlloyd**
Hi Again!

I was trying to use the current Nugert version of SimpleAuth (1.18) with my project and just add this DropBox provider in the source in a PCL common auth lib. However, doing that proved problematic, as it compiled against the SimpleAuth\System.Web.HttpUtility which ParseQueryString (HttpValueCollection) has no BCL equivalent...

Hence Changed its signature to match BCL and added TypeForward attribute for better PCL support

### CODE REVIEW

**Comment by Clancey** [comment]
This commit pull isn't mergable.  Any way you can pull the latest master in, and fix the merge conflicts?

**Comment by superlloyd** [comment]
My GOT-fu is not that good... 
But if I can't fix merge issue.. I might be able to do another pull request...
Trying now...

**Comment by superlloyd** [comment]
using TortoiseGit I did rebase on my master branch with yours, it looks like the intended result....
And now the code can be merged!!

As a side note... the code in the SimpleAuth.Portable library is misleading, since it never SHOULD be used... (intsead SinpleAuth.Android or SimpleAuth.iOS should be used) You can safely replace the medthod implementation by something like "throw new PlatformNorSupportedException("PCL")" ;)

File: src/SimpleAuth/Providers/DropBox.cs [hunk 1]
```diff
@@ -12,7 +13,7 @@ public class DropBoxAuth : OAuthApi
         public DropBoxAuth(string client_id, string client_secret, HttpMessageHandler handler = null)
             : base("DropBox", client_id, client_secret, handler)
         {
-			this.ScopesRequired = false;
+            this.Scopes = new[] { "???" };
```
Comment (Clancey) [inline src/SimpleAuth/Providers/DropBox.cs]: So this change should be put back now that Scopes required has been added.

**Comment by Clancey** [comment]
Yes, the PCL dll should never be used, but I was eventually planning on making it work. Which is why I stubbed most things out rather than not supported exceptions. 

Also as you noticed you should pull the latest master before sending a pull request. It was confused since I already merged in those changes from your last pull request.  Thanks again for this pull request.

**Comment by superlloyd** [comment]
No worries... didn't notice any TypeForwardAttribute and thought you might be confused, haha!

Learning advanced Github usage hey! :)
And you're welcome! :)

**Comment by Clancey** [comment]
Did you see this comment? https://github.com/Clancey/SimpleAuth/pull/7/files#r64267062

I think its the only update missing!

**Comment by superlloyd** [comment]
Done!
Thanks for your patience! ;)

**Pull request merged**